### ISSUE TITLE
Improve Accessibility score by adding [lang] attribute to <html> element for Settings -> Two-factor authentication page 

### ISSUE THREAD

**Issue by riajots** [CMU-313/Teedy#47]
Currently the accessibility score for this page is 89, and one of the issues is that the <html> element does not have a specified [lang] attribute, which could cause the screen reader to not announce the text correctly if the text is not written in the default language.

The goal is to specify the language to ensure this doesn't occur.

**Comment by riajots** [comment]
I would like to work on this please!
